File: Beginner/GameTime_1046/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	values := bufio.NewReader(os.Stdin)
	line, _ := values.ReadString('\n')
	line = strings.Trim(line, "\n")
	valuesSplitted := strings.Split(line, " ")

	initialHour, _ := strconv.ParseInt(valuesSplitted[0], 64, 10)
	endHour, _ := strconv.ParseInt(valuesSplitted[1], 64, 10)

	duration := 24

	if endHour > initialHour {
		duration = int(endHour) - int(initialHour)
	} else if endHour < initialHour {
		duration = 24 + int(endHour) - int(initialHour)
	}
	fmt.Printf("O JOGO DUROU %d HORA(S)\n", duration)

}

File: Beginner/GameTime_1046/main.py
inputted_values = list(map(int, input().split(" ")))

start_hour = inputted_values[0]
end_hour = inputted_values[1]
duration = 24

if end_hour > start_hour:
    duration = end_hour - start_hour
elif end_hour < start_hour:
    duration = 24 + end_hour - start_hour

print(f'O JOGO DUROU {duration} HORA(S)')
